tests/test_core/test_decorator_calls.py

```diff
@@ -25,33 +33,37 @@ def test_basic_decorator():
     list(conn.get_all_instances()).should.equal([])
 
 
-def test_context_manager():
+def test_context_manager(*args):
     conn = boto.connect_ec2('the_key', 'the_secret')
-    with assert_raises(EC2ResponseError):
-        conn.get_all_instances()
+    with mock.patch('boto.connection.AWSQueryConnection.make_request', return_value=MockHttpResponse()):
```

Comment:
What was happening before adding these patches? I'm trying to figure out why they are needed.